pyscriptjs/src/interpreter_worker/worker.ts

```diff
@@ -0,0 +1,26 @@
+// XXX: what about code duplications?
+// I think that with the current build configuration, the code for logger,
+// remote_interpreter and everything which is included from there is
+// bundled/fetched/executed twice, one in pyscript.js and one in
+// worker_interpreter.js. Is this true?
+
+import { getLogger } from '../logger';
+import { RemoteInterpreter } from '../remote_interpreter';
+import * as Synclink from 'synclink';
+
+const logger = getLogger('worker');
+logger.info('Interpreter worker starting...');
+
+async function worker_initialize(cfg) {
+    const remote_interpreter = new RemoteInterpreter(cfg.src);
+    // this is the equivalent of await import(interpreterURL)
+    logger.info(`Downloading ${cfg.name}...`); // XXX we should use logStatus
+    importScripts(cfg.src);
```

Comment:
Eventually we might make `RemoteInterpreter` responsible for loading, though doing it here is economical. Have to remember that `RemoteInterpreter` won't need it until `loadInterpreter` is called.

Reply:
> Eventually we might make RemoteInterpreter responsible for loading,
yes, I would like to do that.
And also, if I understand correctly if/when we can switch to using a ES module for loading pyodide we will be able to unify the two different code paths